scripts/us_census/acs5yr/subject_tables/common/resolve_geo_id_test.py

```diff
@@ -0,0 +1,41 @@
+"""Tests for resolve_geo_id."""
+
+import unittest
+from resolve_geo_id import _convert_to_geoId
+
+
+class ResolveGeoIdTest(unittest.TestCase):
```

Comment:
Test for country?

Reply:
Is it really done? Not seeing a test for "country/USA"

Reply:
Thank you the review.

- I had the country test but missed checking it in the last-time around. The [test](https://github.com/datacommonsorg/data/pull/513#issuecomment-925981114) is included now.
- I have also fixed the `ModuleNotFoundError` issue while running unit tests with `python3 -m unittest discover -v -s . -p "*_test.py"` command. I had not been running with this command instead did `python3 resolve_geo_id_test.py` and hence this error slipped through.

Reply:
Testing guidelines: https://github.com/datacommonsorg/data#testing